djangocms_pageadmin/admin.py

```diff
@@ -165,9 +165,14 @@ def get_list_actions(self):
         ]
 
     def _get_preview_link(self, obj, request, disabled=False):
+
         return render_to_string(
             "djangocms_pageadmin/admin/icons/preview.html",
-            {"url": get_object_preview_url(obj), "disabled": disabled, "keepsideframe": False},
+            {
+                "url": "{}?live-URL={}".format(get_object_preview_url(obj), _get_url(obj)),
```

Comment:
The parameter for 'live-url' should be a setting, this is consistent with the core cms.